ambari/administration/overview.rst

```diff
@@ -0,0 +1,101 @@
+Administering Ambari: Overview
+==============================
+
+Apache Ambari is a system to help you provision, manage and monitor |fdd| clusters. This guide is intended for Cluster Operators and System Administrators responsible for installing and maintaining Ambari and the |fdd| clusters managed by Ambari. Installing Ambari creates a default user with "Admin Admin" privilege, with the following username/password: `admin/admin`.
```

Comment:
I think "Ambari Admin" is more correct.
"Ambari Ambari" is the name + surname of the user created during installation.